Documentation/teaching/so2/assign2-driver-uart.rst

```diff
@@ -0,0 +1,163 @@
+==========================
+Assignment 2 - Driver UART
+==========================
+
+-  Deadline: :command:`Wednesday, 21 April 2021, 23:00`
```

Comment:
April 21st

Reply:
Also, add a note mentioning that this assignment can be made in teams (max 2). Only one of them must submit the assignment, and the names of the student should be listed in a README file.